Makefile

```diff
@@ -184,8 +184,8 @@ ALL_GEN_HEADERS += gen_version.h
 
 CDUMP_OBJS := ccan-cdump.o ccan-strmap.o
 
-WIRE_GEN := tools/generate-wire.py
-BOLT_GEN := tools/generate-bolts.py
+BOLT_GEN := tools/generate-wire.py
+WIRE_GEN := $(BOLT_GEN)
```

Comment:
call me weird, but i kind of like how we've now clearly 'labeled' which csv's are 'bolts' and which are 'wires'. it's really an unnecessary distinction since the formatter handles them the same.